draft-ietf-taps-arch.md

```diff
@@ -329,7 +329,21 @@ The Transport System Implementation Concepts define the set of objects used inte
 
 * Path Selection: Path Selection represents the act of choosing one or more paths that are available to use based on the Path Selection Properties provided by the application, and a Transport Services system's policies and heuristics.
 
-* Protocol Selection: Protocol Selection represents the act of choosing one or more sets of protocol options that are available to use based on the Protocol Properties provided by the application, and a Transport Services system's policies and heuristics.
+* Protocol Selection: Protocol Selection represents the act of choosing one or more sets of protocol options that are available to use based on the Protocol Properties provided by the application, and a Transport Services system's policies and heuristics. See {{security-equivalence}} for considerations regarding security Protocol Selection.
+
+#### Security Protocol Gathering and Equivalence {#security-equivalence}
+
+In some cases, a Transport Services system's policies and heuristics, or individual 
+Protocol Properties, heuristics may yield multiple candidate security protocols. For example, 
+both DTLS and TLS may be viable if an application does not express options that
+are prohibited by either UDP or TCP. In such cases, only one security protocol
```

Comment:
What do you mean by "only one security protocol SHOULD be used"? Also, we aren't using capital SHOULDs and MUSTs right now in the document.

Reply:
Don't try to race DTLS and QUIC!

Reply:
Okay, that's not very clear